docs/usage.md

```diff
@@ -59,17 +56,17 @@ Note that this must only be included in one source file per DSO.
 
 ### Sharing data
 
-Some statically-allocated data must be made accessible to all compartments to
-avoid significant refactoring. In these cases `IA2_SHARED_DATA` can be used to
-place variables in a special ELF section which is explicitly accessible from all
+Some non-`const` statically-allocated data must be made accessible to all compartments to
+avoid significant refactoring. In these cases, `IA2_SHARED_DATA` can be used to
+place variables in a special ELF section that is explicitly accessible from all
```

Comment:
I think "which" is actually more appropriate here: there are no other such ELF sections.